.eslintrc.json

```diff
@@ -0,0 +1,69 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "installedESLint": true,
+    "parserOptions": {
+        "ecmaFeatures": {
+            "experimentalObjectRestSpread": true
+        }
+    },
+    "plugins": [
+        "mozilla"
+    ],
+    "globals": {
+        "Components": false,
+        "Services": false,
+        "Cc": false,
+        "Ci": false,
+        "VTTabDataStore": false,
+        "VerticalTabs": false,
```

Comment:
So, I think these two should probably be on [a per-file basis](http://eslint.org/docs/user-guide/configuring#specifying-globals), since they get imported into each file they're used in…